cmd/edit/cluster/cmd.go

```diff
@@ -397,6 +397,10 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	if len(noProxySlice) > 0 {
+		if len(noProxySlice) == 1 && noProxySlice[0] == doubleQuotesToRemove {
+			noProxySlice[0] = ""
+		}
+
```

Comment:
shouldn't we treat `""` the same as an empty slice?

Reply:
no. we need to explicitly treat "" to remove the current value.

Reply:
Ah, good point.